### ISSUE TITLE
I need not only test accuracy  but test  loss also...

### ISSUE THREAD

**Issue by zeek-han** [clovaai/voxceleb_trainer#45]
When i train with default setting, VEER is calculated for every 10 iteration, and I regard VEER as Validation Equal Error Rate.
thus, VEER is derived from test accuracy and it is very important measure.
But I need not only VEER but Validation Loss also to pick the best model.
How could I print Validation loss?

Thank you...

**Comment by joonson** [comment]
There is no 'validation loss' as such, since the train function is not run on the validation set.
To implement this function, you will need to:
1. Initiate data loader for the validation set
2. Modify `train_network` function to run with `self.eval()` mode.
3. Run this function without the weight updates.

**Comment by zeek-han** [comment]
Oh, I think I express improper way thus you misunderstood my purpose...
The sample code you give just train on TRAINING SET, and for every 10 epoch, calculate EER on test accuracy (VEER) and print it.

**Comment by joonson** [comment]
I think my answer still remains the same. You will need to run the train function without the weight update every 10 epochs. Currently the script does not compute the validation loss.

**Comment by zeek-han** [comment]
Oh, thank you for your answer! I understand it!
